src/Microsoft.AspNetCore/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "1.0.1-*",
```

Comment:
Is this for the 1.0.0 branch? In which case, this would go in the 1.0.0 branch and you'd have to use the KoreBuild from one of the master branches of other repos (it pins versions)